### PR TITLE
Update API test_permission module

### PR DESCRIPTION
**Pull request by elyezer**
Remove SCAP permission entries if the plugin is not installed.

I've run on RHEL7 with the SCAP plugin installed and got the following:

```
$ py.test tests/foreman/api/test_permission.py
========================== test session starts ===========================
platform linux -- Python 3.4.2, pytest-2.9.1, py-1.4.31, pluggy-0.3.1
rootdir: /home/elyezer/code/robottelo, inifile: 
plugins: cov-2.2.1, testmon-0.8.2, xdist-1.14
collected 7 items 

tests/foreman/api/test_permission.py .......

======================= 7 passed in 318.37 seconds =======================
```

On RHEL6 the test is failing but due to other reasons, but the SCAP related permissions are updated properly:

```
E           AssertionError: {
E            "removed_names": [
E             "execute_template_invocation",
E             "view_job_templates",
E             "view_job_invocations",
E             "destroy_job_templates",
E             "filter_autocompletion_for_template_invocation",
E             "create_job_templates",
E             "lock_job_templates",
E             "edit_job_templates",
E             "create_job_invocations",
E             "edit_remote_execution_features"
E            ],
E            "removed_resource_types": [
E             "RemoteExecutionFeature",
E             "JobTemplate",
E             "TemplateInvocation",
E             "JobInvocation"
E            ]
E           }
```

### CODE REVIEW

File: robottelo/constants.py [hunk 1]
```diff
@@ -479,17 +470,17 @@
         'edit_recurring_logics',
     ],
     'ForemanOpenscap::Policy': [
+        'assign_policies',
```
Comment (elyezer) [inline robottelo/constants.py]: Just alphabetical sorting

File: robottelo/constants.py [hunk 2]
```diff
@@ -479,17 +470,17 @@
...
+        'create_policies',
         'destroy_policies',
         'edit_policies',
         'view_policies',
-        'create_policies',
-        'assign_policies',
     ],
     'ForemanOpenscap::ScapContent': [
-        'destroy_scap_contents',
```
Comment (elyezer) [inline robottelo/constants.py]: Just alphabetical sorting

File: tests/foreman/api/test_permission.py [hunk 3]
```diff
@@ -34,6 +35,14 @@ def setUpClass(cls):
             cls.permissions[None].remove('view_log_viewer')
             cls.permissions[None].remove('view_search')
 
+        result = ssh.command('rpm -qa | grep rubygem-foreman_openscap')
+        if result.return_code != 0:
+            cls.permissions.pop('ForemanOpenscap::Policy')
+            cls.permissions.pop('ForemanOpenscap::ScapContent')
+            cls.permissions[None].remove('destroy_arf_reports')
```
Comment (omaciel) [inline tests/foreman/api/test_permission.py]: None?

Comment (elyezer) [inline tests/foreman/api/test_permission.py]: Yes, Python allows any object being the key of a dictionary and we use None to indicate that a permission is not related to any specific resource. We can call that the way we map global permissions.

Comment (omaciel) [inline tests/foreman/api/test_permission.py]: Neat-o. It helped me to understand it better when I realized that `permissions[None]` contained a `list` :)

```python
In [6]: foo = {'a': 1, 'b': 2, None: ['c', 'd', 'e']}

In [7]: foo[None].remove('c')

In [8]: foo
Out[8]: {None: ['d', 'e'], 'a': 1, 'b': 2}
```

**Comment by omaciel** [comment]
ACK and roll
